### DISCUSSION TITLE
read the message sent by the user

### DISCUSSION THREAD

**Discussion by ah-rahimi** [def-studio/telegraph#121] [Q&A] [answered]
Hello
I did not understand how I can read the message sent by the user and respond to the user according to the message
I read here:
https://def-studio.github.io/telegraph/features/messages
But it was only about commands and Callback Queries and nothing was said about the text messages
please help

**Comment by ah-rahimi** [comment]
@fabio-ivona

**Comment by fabio-ivona** [comment, marked as answer]
@ahr1384 you can follow this discussion for more info:

https://github.com/def-studio/telegraph/discussions/119#discussioncomment-3162578
